### ISSUE TITLE
input styles and output styles could be separate

### ISSUE THREAD

**Issue by ajlee2006** [timeopochin/tokiponakeyboard#14]
this could be a setting hidden in some "advanced" section because idk if people would use it a lot but it would be cool if for example, have a chinese keyboard and output sitelen emoji. or something. or have sitelen emoji keyboard and output Latin text.

that said, to add on, Latin text could have two kinds of outputs, one that would work well for linja pona font and one for normal typing. for example, "mi jan [_alasa_ni_taso_ilo_lape_utala] li toki e toki-pona" for linja pona, but "mi jan Antilu li toki e toki pona" for normal typing"??? should this be a different post

**Comment by neonpixii** [comment]
i agree; i use this keyboard for just normal latin toki pona input, and i think having a way to type latin letters would be good

the "wetuiop" keyboard layout i've seen proposed for toki pona goes
```
        wetuiop
[shift] asnmjkl [bksp]
[num] , [space] . [ret]
```

the remaining letters of the latin alphabet (bcdfghqrvwxyz) are accessible by long pressing on their corresponding toki pona letters and selecting them from a popup menu 
```
w → v
t → d
p → b f h 
s → c x z
j → y 
k → g q
l → r
```

with symbols such as ? ' " : ; ! @ # - + $ positioned on the remaining keys as long press menus. the keyboard proposal i like uses
```
e → ?
u → '
i → "
o → : ;
a → !  @ (as in toki pona "a")
n → # (as in nanpa)
m → - + $ (as in mani)
```

i would love to be able to hit a key on the sitelen pona keyboard to get a wetuiop keyboard like this to type out a name real quick before going back to the main sitelen pona keyboard
